gstreamer-rtsp-server/src/auto/r_t_s_p_media_factory_u_r_i.rs

```diff
@@ -0,0 +1,118 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir @ 9ae0a4a+)
+// from gir-files (https://github.com/gtk-rs/gir-files @ ???)
+// DO NOT EDIT
+
+use RTSPMediaFactory;
+use ffi;
+use glib;
+use glib::StaticType;
+use glib::Value;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::signal::SignalHandlerId;
+use glib::signal::connect;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::boxed::Box as Box_;
+use std::mem;
+use std::mem::transmute;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct RTSPMediaFactoryURI(Object<ffi::GstRTSPMediaFactoryURI, ffi::GstRTSPMediaFactoryURIClass>): RTSPMediaFactory;
+
+    match fn {
+        get_type => || ffi::gst_rtsp_media_factory_uri_get_type(),
+    }
+}
+
+impl RTSPMediaFactoryURI {
+    pub fn new() -> RTSPMediaFactoryURI {
+        assert_initialized_main_thread!();
+        unsafe {
+            from_glib_full(ffi::gst_rtsp_media_factory_uri_new())
+        }
+    }
+}
+
+impl Default for RTSPMediaFactoryURI {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+unsafe impl Send for RTSPMediaFactoryURI {}
+unsafe impl Sync for RTSPMediaFactoryURI {}
+
+pub trait RTSPMediaFactoryURIExt {
+    fn get_uri(&self) -> Option<String>;
+
+    fn set_uri(&self, uri: &str);
```

Comment:
Can be NULL, or not? Getter and setter disagree

Reply:
legit though :(